src/util/snippets.py

```diff
@@ -322,6 +329,8 @@ def convert_html_content(
     :param dict kwargs: extra keyword arguments to pass to :func:`convert_html_column`
     """
 
+    if "set_format" in dir(converter_callback):
```

Comment:
Do we need this `if` ?

Reply:
Since `upgrade-util` is an open repo, it might be used with a `converter_callback` that is not the expected `HTMLConverter` from this class.